projects/control-service/projects/pipelines_control_service/src/main/resources/application-prod.properties

```diff
@@ -131,4 +131,4 @@ datajobs.monitoring.sync.initial.delay=${DEFAULT_MONITORING_INITIAL_DELAY_SYNC_I
 #datajobs.deployment.builder.extraArgs=${DATAJOBS_DEPLOYMENT_BUILDER_EXTRAARGS}
 
 # Enables JSON-formatted logging for all logs emitted by the Control Service.
-# LOGGING_FORMAT=JSON
+LOGGING_FORMAT=${CONTROL_SERVICE_LOGGING_FORMAT:""}
```

Comment:
pipelinesControlService is redundant. For other control service options, we do not set a prefix. As it's a pipelines control service chart it's implied. For example, we have "notificationOwnerName" and not "controlServiceOwnerName" . "loggingFormat" or "serviceLoggingFormat" is enough

Reply:
gotcha, I'll change it